_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,294 @@
+
+In this article, I’ll explain how to deal with the simple requirement of running a Laravel application as a local Kubernetes set up.
+
+## Kubernetes, Why and What?
+
+Kubernetes is an open-source system initially designed by Google to facilitate the management of containerised applications in a clustered environment. Some refer to it as an orchestration platform, and Kubernetes is not the only available platform of this type. Although, it does have a very high and fast adoption rate. Not to mention that it is quite easy to implement once you get used to it.
+
+If you're still wondering why someone can benefit from using Kubernetes, the answer is simple. Kubernetes makes it easier to set up and manage as many clusters as required across multiple projects.
+
+## Deploying a Laravel Application to Minikube
+
+As I've afore-mentioned, I will be showing you how to deploy a straightforward and stateless Laravel application to Kubernetes. I aim to detail the steps involved in accomplishing this while explaining why specific actions are required. Furthermore, I will show you how to quickly scale the application and also make it available on a named host using an Ingress controller.
+
+You can run Kubernetes on several cloud hosting providers such as Google Cloud Engine and Amazon Web Services. In this tutorial, you will run the application on Minikube, a tool that makes it easy to run Kubernetes locally.
+
+Similar to Vagrant, Minikube is merely a Virtual Machine that contains a Kubernetes platform and Docker. You will need both to deploy your application as a Docker container and scale it to three instances using Kubernetes.
+
+
+__The application__
+
+I have prepared a simple Laravel application which you can clone from <a href="https://github.com/learnk8s/laravel-kubernetes-demo" target="_blank" rel="noopener" class="link">GitHub</a>. It is nothing more than a fresh Laravel installation. Therefore you can follow this example using either the demo application or just create a new Laravel application. To use the demo application clone it in your project's directory.
+
+```bash
+cd /to/your/working/directory
+git clone git@github.com:learnk8s/laravel-kubernetes-demo.git .
+```
+
+__Prerequisites__
+
+To follow with this demonstration, you will need the following installed on your local system:
+
+
+1) <a href="https://docs.docker.com/install/" target="_blank" rel="noopener">Docker</a>
+
+2) <a href="https://kubernetes.io/docs/tasks/tools/install-kubectl/" target="_blank" rel="noopener">Kubectl</a>
+
+3) <a href="https://github.com/kubernetes/minikube/releases" target="_blank" rel="noopener">Minikube</a>
+
+_Are you having problems installing and running these applications on Windows? LearnK8s will be publishing a tutorial on how to accomplish this very soon. Subscribe and be notified as soon as new articles get published._
```

Comment:
Perhaps we could have this as a blockquote?

Reply:
👍